### ISSUE TITLE
Weapon Wield Crash

### ISSUE THREAD

**Issue by qwertydude567** [CleverRaven/Cataclysm-DDA#28328]
**Describe the bug**  
The game can't seem to handle wielding (I tried a bunch of different weapons/tools) and seems to crash upon the second wielding (wield [debug report], unwield, 2nd wield brings game crash).

**To Reproduce**  
1. I loaded a saved game made prior upgrading to the newest version.  Possibly ~5 builds ago... Not sure.

2. If I try to wield a weapon (I tried 5 so far: entrenching tool, combat knife, hunting knife, katana, and [modified] composite crossbow), I get a debug report (as an example, here is the entrenching tool one):
19:35:20.337 : Starting log.
19:35:20.337 INFO : Cataclysm DDA version 0.C-37555-gf76242f
19:35:20.404 INFO : Number of render drivers on your system: 4
19:35:20.404 INFO : Render driver: 0/direct3d
19:35:20.405 INFO : Render driver: 1/direct3d11
19:35:20.405 INFO : Render driver: 2/opengl
19:35:20.405 INFO : Render driver: 3/software
19:35:20.416 INFO : Language is set to: 'en'
19:35:22.327 INFO : SDL version used during compile is 2.0.5
19:35:22.327 INFO : SDL version used during linking and in runtime is 2.0.5
19:35:22.501 INFO : Active renderer: 3/software
19:35:22.979 INFO : USE_COLOR_MODULATED_TEXTURES is set to 0
19:35:23.303 WARNING : opendir [./mods/] failed with "No such file or directory".
19:35:27.327 WARNING : opendir [./save/Riverland/mods] failed with "No such file or directory".
19:35:36.727 INFO : Loaded tileset: MSX++DEAD_PEOPLE
19:35:43.109 ERROR : (error message will follow backtrace)
	(@0x5eae6e[cataclysm-tiles.exe+0x1eae6e]), 
	(@0x5eb44a[cataclysm-tiles.exe+0x1eb44a]), 
	(@0x5ed0e3[cataclysm-tiles.exe+0x1ed0e3]), 
	(IMG_LoadWEBP_RW+0x111165@0x10aecc5[cataclysm-tiles.exe+0xcaecc5]), 
	(IMG_LoadWEBP_RW+0x2931a2@0x1230d02[cataclysm-tiles.exe+0xe30d02]), 
	(IMG_LoadWEBP_RW+0x2933fa@0x1230f5a[cataclysm-tiles.exe+0xe30f5a]), 
	(IMG_LoadWEBP_RW+0x299882@0x12373e2[cataclysm-tiles.exe+0xe373e2]), 
	(@0x6b5ba2[cataclysm-tiles.exe+0x2b5ba2]), 
	(@0x6b5c51[cataclysm-tiles.exe+0x2b5c51]), 
	(@0x6edeaf[cataclysm-tiles.exe+0x2edeaf]), 
	(@0x70d9ab[cataclysm-tiles.exe+0x30d9ab]), 
	(@0x71ec7d[cataclysm-tiles.exe+0x31ec7d]), 
	(@0x700c16[cataclysm-tiles.exe+0x300c16]), 
	(IMG_LoadWEBP_RW+0x4e8de9@0x1486949[cataclysm-tiles.exe+0x1086949]), 
	(@0x4013ed[cataclysm-tiles.exe+0x13ed]), 
	(@0x4014fb[cataclysm-tiles.exe+0x14fb]), 
	(BaseThreadInitThunk+0x14@0x7fff10ed3dc4[KERNEL32.DLL+0x13dc4]), 
	(RtlUserThreadStart+0x21@0x7fff110d3691[ntdll.dll+0x73691]), 
Backtrace emission took 0 seconds.
src/visitable.cpp:30 [item* visitable<T>::find_parent(const item&) [with T = Character]] Tried to find item parent using an object that doesn't contain it
19:35:48.380 ERROR : src/character.cpp:838 [item Character::i_rem(const item*)] did not found item entrenching tool to remove it!
19:35:53.840 : Log shutdown.

3. I'm not sure exactly how it goes after that, but I do know that I tried this with my [modified] composite crossbow a couple times and I think that afterwards I unwielded, then wielded again and upon the second wield the game locked up my computer so bad that CTR-ALT-DELETE wouldn't even work right and I had to reset my computer by flipping the switch on my case power supply.  I won't even attempt this again.

And perhaps with the combat knife and katana I tried this and just got some crash popup upon the 2nd wield that upon closing the popup closed the game out.  I'd post the text from the crash but I I already started typing this bug report and don't want to suffer the chance that my computer power needs to be reset mid bug report.

**Expected behavior**
The game to not show an error notification or crash.

**Versions and configuration(please complete the following information):**
 - OS: Windows 10
 - Game Version: 0.C-37555-gf76242f (build: 8536, though I'm guessing the issue probably appeared in 8534 based off of the changelog)
 - Graphics version: Tiles
 - Mods loaded: DDA, Filthy Clothing, Extended Realistic Guns, Icecoon's Arsenal, Makeshift Items Mod, More Survival Tools, More Locations, Boats, Folding Parts Pack, Vehicle Additions Pack, Tanks and Other Vehicles, Classes and Scenarios Mod, Classic Roguelike Classes, (DP_RP) REMIX Indicators

**Comment by chaoticidealism** [comment]
This also happens with a newly generated world and a new game.

**Comment by Xelat84** [comment]
Same for me, trying to wield throwing sticks got this message:
``` DEBUG    : Tried to find item parent using an object that doesn't contain it
 
 FUNCTION : item* visitable<T>::find_parent(const item&) [with T = Character]
 FILE     : src/visitable.cpp
 LINE     : 30
```

**Comment by ifreund** [comment]
Fixed by #28318

**Comment by qwertydude567** [comment]
Thanks - I confirm.  No more issue with it that I was seeing myself.
